### PR TITLE
#236 Add Lightbend SBR with Keep Majority enabled

### PR DESCRIPTION
**Pull request by edwardcallahan**
Enable Lightbend SBR with the Keep Majority strategy enabled.

### CODE REVIEW

File: build.sbt [hunk 1]
```diff
@@ -82,6 +83,9 @@ def akkaDiscoveryDependency(name: String, excludeThese: ExclusionRule*) =
 def akkaPersistenceCassandraDependency(name: String, excludeThese: ExclusionRule*) =
   "com.typesafe.akka" %% name % AkkaPersistenceCassandraVersion excludeAll ((excludeTheseDependencies ++ excludeThese): _*)
 
+def akkaSBRDependency(name: String, excludeThese: ExclusionRule*) =
```
Comment (ihostage) [inline build.sbt:86]:
```suggestion
def akkaEnhancementsDependency(name: String, excludeThese: ExclusionRule*) =
```
How about `akkaEnhancementsDependency` if the name of the module is a parameter?

File: build.sbt [hunk 2]
```diff
@@ -54,6 +54,7 @@ val AkkaVersion = "2.6.5"
 val AkkaHttpVersion = "10.1.11"
 val AkkaManagementVersion = "1.0.5"
 val AkkaPersistenceCassandraVersion = "0.102"
+val AkkaSBRVersion = "1.1.14"
```
Comment (ihostage) [inline build.sbt:57]:
```suggestion
val AkkaEnhancementsVersion = "1.1.14"
```

**Comment by sleipnir** [comment]
> Commercial module in the open-source project. It's very cool. But it's legal? smile

https://www.lightbend.com/blog/lightbend-to-contribute-commercial-features-to-akka-open-source

File: build.sbt (hunk 1 above)
Comment (edwardcallahan) [reply build.sbt:86]: Thanks for the suggestion!

**Comment by edwardcallahan** [comment]
It is now :)
https://www.lightbend.com/blog/lightbend-to-contribute-commercial-features-to-akka-open-source

**Comment by edwardcallahan** [comment]
Ah. "we will integrate Lightbend’s Split Brain Resolver into the core of Akka Cluster".
So we should wait for the next Akka.

**Pull request closed without merging**